packages/react-day-picker/src/contexts/Focus/getInitialTargetFocusedDay.tsx

```diff
@@ -0,0 +1,38 @@
+import { addDays, endOfMonth, startOfMonth } from 'date-fns';
+
+import { getModifierStatus } from 'hooks/useDayModifiers/utils/getModifierStatus';
+import { Modifiers } from 'types/Modifiers';
+
+export function getInitialTargetFocusedDay(
```

Comment:
We need the jsdocs here. I am writing it right?

```suggestion
/** Returns the day that should be target of the focus when initially rendered according to... */
export function getInitialFocusTarget(
```